memegen/services/image.py

```diff
@@ -22,6 +23,13 @@ def find_template(self, key):
     def create_image(self, template, text, style=None):
         image = Image(template, text, style=style)
 
+        data = {
+            'path': text.path,
+            'template': template.key,
+            'style': style,
+        }
+        redis.hmset('latest', data)
```

Comment:
Thats probably a good idea. I just thought of a way to put all the redis stuff in the store, but I have to play with it some more...